scripts/genproto.sh

```diff
@@ -15,41 +15,32 @@ if ! [[ $(protoc --version) =~ "3.5.1" ]]; then
 	exit 255
 fi
 
-# Exact version of plugins to build.
-PROTOC_GEN_GOGOFAST_SHA="971cbfd2e72b513a28c74af7462aee0800248d69"
-PROTOC_GEN_GRPC_ECOSYSTEM_SHA="e4b8a938efae14de11fd97311e873e989896348c"
-
 echo "installing plugins"
-go install "golang.org/x/tools/cmd/goimports"
-go get -d -u "github.com/gogo/protobuf/protoc-gen-gogo"
-pushd ${GOPATH}/src/github.com/gogo/protobuf
-    git reset --hard "${PROTOC_GEN_GOGOFAST_SHA}"
-    go install "github.com/gogo/protobuf/protoc-gen-gogofast"
-popd
+INSTALL_PKGS="golang.org/x/tools/cmd/goimports github.com/gogo/protobuf/protoc-gen-gogofast github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger"
+for pkg in ${INSTALL_PKGS}; do
+    GO111MODULE=on go install -mod=vendor "$pkg"
```

Comment:
What will happen if those packages are already installed in wrong version? Not fully familiar with go mod.. does it helps here?